clients/src/main/resources/common/message/DescribeProducersRequest.json

```diff
@@ -27,6 +27,5 @@
         { "name": "PartitionIndexes", "type": "[]int32", "versions": "0+",
           "about": "The indexes of the partitions to list producers for." }
        ]}
-    ]}
```

Comment:
Good catch. Could we re-indent the `Topics` (L24) block now that we have removed the trailing json? We usually two spaces.

Reply:
1e9d680e4225de93169cb357147723d7527b3503 🎉